packages/module/src/ChatbotConversationHistoryNav/ChatbotConversationHistoryDropdown.tsx

```diff
@@ -19,13 +19,16 @@ export interface ChatbotConversationHistoryDropdownProps extends Omit<DropdownPr
   label?: string;
   /** Callback for when user selects item. */
   onSelect?: (event?: React.MouseEvent, value?: string | number) => void;
+  /** Id applied to dropdown */
```

Comment:
In case we expose dropdown props to pass an ID to the dropdown itself, differentiate it from the toggle ID:

```suggestion
  /** Id applied to dropdown menu toggle */
```